client/package.json

```diff
@@ -29,6 +29,7 @@
     "style-loader": "^1.2.1",
     "vue": "^2.6.11",
     "vue-contenteditable": "^1.0.2",
+    "vue-hotjar": "^1.2.0",
```

Comment:
Please also update app version by 0.0.1.

Reply:
Version bump.

Reply:
For some reason, the version in the package-lock file did not get updated. Please update manually or do `npm i` again.

Reply:
Fixed.